src/pagination/pagination.js

```diff
@@ -175,33 +211,43 @@ export default class Pagination extends React.PureComponent<
               </Button>
               {isMenuOpen && (
                 // $FlowFixMe
-                <DropdownMenu
-                  items={options}
-                  onItemSelect={this.onMenuItemSelect}
-                  initialState={{
-                    highlightedIndex: Math.max(currentPage - 1, 0),
-                  }}
-                  overrides={{
-                    List: {
-                      component: StyledDropdownMenu,
-                      // Access $style manually because it has gone through transformation
-                      // from the override helper function already
-                      // $FlowFixMe
-                      style: dropdownMenuProps.$style,
-                    },
-                  }}
-                  {...dropdownMenuProps}
-                />
+                <div role="menu">
```

Comment:
The [dropdown menu](https://github.com/uber-web/baseui/blob/master/src/menu/menu.js#L31) should have a `listbox` role. Is `menu` more appropriate? If so, we try to allow consumers of baseui to customize all 'sub-components'. can the role be passed to `List` override?

```jsx
<DropdownMenu
  overrides={{
    List: {
      props: {role: 'menu'},
    },
  }}
/>
```

Reply:
Thanks @chasestarr !
Ahh, I see. Good suggestion.. I changed the role to an override and I added an override for the options as well to be `menuitem`.